GANDLF/data/preprocessing/normalize_rgb.py

```diff
@@ -15,7 +15,9 @@ class NormalizeRGB(IntensityTransform):
 
     """
 
-    def __init__(self, mean: list = None, std: list = None, **kwargs):
+    def __init__(
+        self, mean: Optional[list] = None, std: Optional[list] = None, **kwargs
```

Comment:
list shoud have type annotation for the items contained

Reply:
Fixed.